src/bin/ion/commands/beta/generate/templates/java/class.templ

```diff
@@ -3,10 +3,10 @@ import ion_data_model.{{ import.name | upper_camel }};
 {% endif %}
```

Comment:
Yes, that is what I intend to do here. Similar to the `struct.templ` imports were actually used to put a reference to anonymous types that are used for current class/struct but if we go ahead with an approach where we don't generate a module `ion_data_model`., we don't have to worry about the imports.

Reply:
There needs to be a package because it's generally bad practice to use the default package. How are we determining the package structure for the generated java classes? (This is something that could be punted for now, though.)

Reply:
Sorry I didn't understand what you meant by there needs to be a package? Are we talking about the imports or the namespace declaration at the beginning like `package ion_data_model`.
Currently we assume the module is at the root of project, but I do intend to make it  dynamic: https://github.com/amazon-ion/ion-cli/issues/70.

Reply:
The namespace declaration. We don't want it to be the default package (because that's considered bad practice) and users of the generated code cannot easily relocate the classes the way one can with rust code. We also cannot use any hard-coded namespace because if two generated models ever end up in the same classpath, and both have e.g. an `UUID` type, then one of the two definitions will clobber the other one at runtime. (Or if you have good static analysis tools, it will be detected and your build will fail.)

There needs to be a strategy for allowing the user to provide a namespace to use as the root namespace of the model, and for namespace-ing types from different schema files (from the same authority) so that they don't collide with each other.

Reply:
Right, which is why to resolve dynamic namespace declaration: #70, I was thinking to add an option with `generate`  like `namespace_path`/`module_path` and that can then be used inside the template to properly declare the namespace. Does this strategy resolve what you are asking here?

Reply:
I've added a comment to the issue that describes more of the problem. I'm fine with deferring it for now.